src/core.c/CompUnit/RepositoryRegistry.pm6

```diff
@@ -76,11 +76,15 @@ class CompUnit::RepositoryRegistry {
                  for parse-include-specS($specs);
             }
 
-            if nqp::existskey($ENV,'RAKUDOLIB') {
+            if nqp::existskey($ENV,'RAKULIB') {
+                nqp::push($raw-specs,$_)
+                  for parse-include-specS(nqp::atkey($ENV,'RAKULIB'));
+            }
+            elsif nqp::existskey($ENV,'RAKUDOLIB') {
```

Comment:
Whatever we decide about the variable, but if it stays – it must be the first, following the principle of most specific case. Wether it should be mutually exclusive with `RAKULIB` or not is still doubtful to me. But considering reasons given by @ugexe, it's better have it non-exclusive for now.

Reply:
Okay, I will make the changes as I understand the comments.